<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackDialog</class>
 <widget class="QDialog" name="PlaybackDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>223</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playback</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QPushButton" name="autoButton">
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="moveLabel">
     <property name="text">
      <string>Click Next or Auto to start playback</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
